src/wp-includes/class-wp-speculation-rules.php

```diff
@@ -230,7 +230,7 @@ public function has_rule( string $mode, string $id ): bool {
 	 *
 	 * @since 6.8.0
 	 *
-	 * @return array<string, array<string, mixed>> Speculation rules data.
+	 * @return array<string, mixed[]> Speculation rules data.
```

Comment:
How about `array<string, array<int, mixed>>`?

Reply:
```suggestion
	 * @return array<string, array<int ,mixed>> Speculation rules data.
```

Reply:
Resolved in a322456d2eb0784f5e4f0dddb195b829f9895ab4

Reply:
The `mixed` here is a rule which is actually an `array<string, mixed>`. I've applied that in f5700b99067f3dee4e004bbe9b749ee8a56b6993.